quarkchain/cluster/master.py

```diff
@@ -1546,11 +1571,15 @@ def main():
         if env.cluster_config.START_SIMULATED_MINING:
             asyncio.ensure_future(master.start_mining())
 
-    if env.cluster_config.use_p2p():
-        network = P2PManager(env, master, loop)
-    else:
-        network = SimpleNetwork(env, master, loop)
-    network.start()
+        if env.cluster_config.CHECK_DB:
+            asyncio.ensure_future(master.check_db())
```

Comment:
how about following to avoid checking multiple flags
```
        if env.cluster_config.CHECK_DB:
              asyncio.ensure_future(master.check_db())
              master.do_loop(None)
              Logger.info("Master server check DB finished")

```

Reply:
Done